package.json

```diff
@@ -16,6 +16,12 @@
   "workspaces": {
     "packages": [
       "packages/*"
+    ],
+    "nohoist": [
```

Comment:
I'm getting a yarn warning telling me that `nohoist` is deprecated. Do we need to update this?

<img width="880" alt="Screen Shot 2022-05-06 at 3 30 53 PM" src="https://user-images.githubusercontent.com/13334214/167224242-8ad15c17-235f-49f3-8ef3-6c2323d75034.png">

Reply:
I downgraded my yarn version because Yarn 2 was causing everything to break, so I'm going to choose to ignore this for now.